testgres/node.py

```diff
@@ -836,7 +847,8 @@ def poll_query_until(self,
                          expected=True,
                          commit=True,
                          raise_programming_error=True,
-                         raise_internal_error=True):
+                         raise_internal_error=True,
+                         zero_rows_is_ok=False):
```

Comment:
In `Subscription.catchup()` we are expecting True and at the same time it may happen that there is no rows at the moment (until statistics collected).